sys/kern/kern_clock.c

```diff
@@ -531,7 +534,10 @@ hardclock(int cnt, int usermode)
 			if (i > 0 && i <= newticks)
 				watchdog_fire();
 		}
-		intr_event_handle(clk_intr_event, NULL);
+#ifdef DEV_ACPI
```

Comment:
Why does this need to be under DEV_ACPI? If it builds and works, just isn't currently doing anything, that's fine.

Reply:
`intr_event_handle()` isn't exactly the cheapest function call.  Partially this seemed since the lack of other uses seems to suggest perhaps a different approach should be used.

The previous comment though is a better reason for dropping from _this_ series.  I'd been bothered by this for a while and it was quite troublesome to track down.